src/components/drivers/pdf-viewer.jsx

```diff
@@ -9,15 +9,23 @@ PDFJS.disableWorker = true;
 const INCREASE_PERCENTAGE = 0.2;
 const DEFAULT_SCALE = 1.1;
 
-class PDFPage extends React.Component {
-
+export class PDFPage extends React.Component {
```

Comment:
can this component be used on its own? why export it?

Reply:
Yes it can be moved to a different file if you want, I export it because I wrote a few few tests for it.  But I didn't move it so that the code changes could be seen more clearly.

Reply:
cool - no it's ok to leave it here